src/index.html

```diff
@@ -17,16 +17,37 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      action="#"
+      data-qa="big"
+      method="get"
+    >
+      <input
+        class="search search--big"
+        name="search-big"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles”"
+        id="big-field"
+      />
+      <label for="big-field"></label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      action="#"
+      method="get"
+      data-qa="small"
+    >
+      <input
+        class="search search--small"
+        name="search-small"
+        type="text"
+        placeholder="Try “Los Angeles”"
+        id="small-field"
+      />
+      <label for="small-field"></label>
```

Comment:
Empty label tags are not good for accessibility. Screen readers use them to read out what the field is supposed to contain to their users. Consider adding a descriptive text inside the label tags.